migration/v0.17.x-v0.18.x/upgrade-apps.md

```diff
@@ -29,6 +29,8 @@
     bin/ck8s init
     ```
 
+1. If you had requests (but not limits) enabled for harbor, you may have to adjust them in your override config to make sure that they are lower than the new limits.
```

Comment:
Is this true only for harbor?

Reply:
It's the only one I noticed a problem for. It's possible that it could happen for other applications too, but I think that you would have had to change the requests manually for it to be a problem (whereas harbor failed with the default requests set in the prod flavor).